### DISCUSSION TITLE
How to disable opening subfolder as root?

### DISCUSSION THREAD

**Discussion by ZeiZel** [nvim-neo-tree/neo-tree.nvim#1595] [Q&A] [answered]
I'm working on a monorepo in NX and I have trouble. When I open a Dockerfile for a microservice, the project subfolder opens as root. Is there a way to stop this from happening?

![image](https://github.com/user-attachments/assets/a172623c-5f3d-4b63-ad9a-a63cb8e088ff)
![image](https://github.com/user-attachments/assets/99d8535d-30f6-474f-94e0-8e19fcc860d6)

Thanks in advance!

**Comment by cseickel** [comment, marked as answer]
What is happening is that you have another plugin that is changing your current working directory based on the file you have opened. You can either remove that other plugin or set neo-tree to not follow the current working directory. See 
```
:help neo-tree-cwd
```
for details.

**Reply by ZeiZel** [reply]
Thanks! It was very helpful! 
Adding `filesystem.bind_to_cwd = false` resolved my problem.
